src/plugins/blocksmith/components/facebook-post-block-modal/getFacebookPostUrl.test.js

```diff
@@ -1,8 +1,19 @@
 import test from 'tape';
 import getFacebookPostUrl from './getFacebookPostUrl';
 
+const isValidUrl = (str) => {
```

Comment:
@hunyoboy if you want to do this, you would want to put it in the util itself, not in the test only. and you can use the `isValidUrl` util from gdbots/common